src/mpid/ch4/src/ch4_globals.c

```diff
@@ -12,6 +12,7 @@
 /* All global ADI data structures need to go in this file */
 /* reference them with externs from other files           */
 
+#include <pmi.h>
```

Comment:
nit: we use `"pmi.h"` everywhere else in the code, including elsewhere in this commit.